lib/ansible/modules/messaging/rabbitmq_binding.py

```diff
@@ -67,6 +67,12 @@
             - rabbitMQ management api port
         required: false
         default: 15672
+    login_with_ssl:
```

Comment:
login_ matches the other parameters that define the host connection, but I think login_tls would be a better name for this one.